### ISSUE TITLE
Add keys from https://github.com/streaak/keyhacks

### ISSUE THREAD

**Issue by BoBeR182** [SpectralOps/keyscope#23]
https://github.com/streaak/keyhacks
Has a list of many ways to test an API Key for validity.

Porting these to KeyScope would be useful.

- [ ]     ABTasty API Key
- [X]     Algolia API key
- [ ]     Amplitude API Keys
- [X]     Asana Access token
- [X]     AWS Access Key ID and Secret
- [ ]     Azure Application Insights APP ID and API Key
- [ ]     Bazaarvoice Passkey
- [ ]     Bing Maps API Key
- [X]     Bit.ly Access token
- [X]     Branch.io Key and Secret
- [X]     BrowserStack Access Key
- [X]     Buildkite Access token
- [ ]     ButterCMS API Key
- [ ]     Calendly API Key
- [X]     CircleCI Access Token
- [ ]     Cypress record key
- [X]     DataDog API key
- [ ]     Delighted API key
- [ ]     Deviant Art Access Token
- [ ]     Deviant Art Secret
- [X]     Dropbox API
- [X]     Facebook Access Token
- [ ]     Facebook AppSecret
- [X]     Firebase
- [ ]     Firebase Cloud Messaging (FCM)
- [ ]     FreshDesk API Key
- [X]     Github client id and client secret
- [ ]     GitHub private SSH key
- [X]     Github Token
- [X]     Gitlab personal access token
- [ ]     GitLab runner registration token
- [ ]     Google Cloud Service Account credentials
- [ ]     Google Maps API key
- [ ]     Google Recaptcha key
- [ ]     Help Scout OAUTH
- [X]     Heroku API key
- [ ]     HubSpot API key
- [ ]     Infura API key
- [ ]     Instagram Access Token
- [ ]     Instagram Basic Display API
- [ ]     Instagram Graph API
- [ ]     Ipstack API Key
- [ ]     Iterable API Key
- [X]     JumpCloud API Key
- [ ]     Keen.io API Key
- [ ]     LinkedIn OAUTH
- [ ]     Lokalise API Key
- [ ]     Loqate API Key
- [X]     MailChimp API Key
- [X]     MailGun Private Key
- [ ]     Mapbox API key
- [ ]     Microsoft Azure Tenant
- [ ]     Microsoft Shared Access Signatures (SAS)
- [ ]     New Relic Personal API Key (NerdGraph)
- [ ]     New Relic REST API
- [ ]     NPM token
- [ ]     OpsGenie API Key
- [X]     Pagerduty API token
- [ ]     Paypal client id and secret key
- [ ]     Pendo Integration Key
- [ ]     PivotalTracker API Token
- [ ]     Razorpay API key and secret key
- [X]     Salesforce API key
- [X]     SauceLabs Username and access Key
- [X]     SendGrid API Token
- [ ]     Shodan.io
- [X]     Slack API token
- [X]     Slack Webhook
- [ ]     Sonarcloud
- [ ]     Spotify Access Token
- [X]     Square
- [X]     Stripe Live Token
- [X]     Travis CI API token
- [X]     Twilio Account_sid and Auth token
- [X]     Twitter API Secret
- [ ]     Twitter Bearer token
- [ ]     Visual Studio App Center API Token
- [ ]     WakaTime API Key
- [ ]     WeGlot Api Key
- [ ]     WPEngine API Key
- [ ]     YouTube API Key
- [ ]     Zapier Webhook Token
- [ ]     Zendesk Access token
- [X]     Zendesk API key

**Comment by Tamer4214** [comment]
712f39db67e6e30cd47472cfd2f938b0fd8e8559b873c92109c248b4c4d393da5c2aa176441d30a30ca40bcd35efb99ff10910b0570c786a1f39c223734d4bd0

**Comment by BoBeR182** [comment]
https://github.com/SpectralOps/keyscope/pull/26
https://github.com/SpectralOps/keyscope/pull/25
updated TODO list.

**Comment by BoBeR182** [comment]
https://github.com/SpectralOps/keyscope/pull/28

**Comment by BoBeR182** [comment]
https://github.com/SpectralOps/keyscope/pull/29 buttercms

**Comment by BoBeR182** [comment]
https://github.com/SpectralOps/keyscope/pull/30 calendly

**Comment by jondot** [comment]
@BoBeR182 thanks!!
If you'd like, we can send you a SpectralOps t-shirt ;), feel free to send me an email: dotan@spectralops.io

**Comment by BoBeR182** [comment]
https://github.com/SpectralOps/keyscope/pull/31 shodan

**Comment by BoBeR182** [comment]
https://github.com/SpectralOps/keyscope/pull/32 pendo

**Comment by BoBeR182** [comment]
@jondot I will continue to contribute the detectors over the upcoming weeks, do you have a roadmap on when to push for a new release?

**Comment by jondot** [comment]
yep I can create a new release early next week, and then no problem to create a new release when ever we need

**Comment by BoBeR182** [comment]
I will leave it to the SpectralOps team to handle release management and just continue with merge requests.

**Comment by jondot** [comment]
yep sounds good!

**Comment by BoBeR182** [comment]
https://github.com/SpectralOps/keyscope/pull/33 hubspot

**Comment by BoBeR182** [comment]
https://github.com/SpectralOps/keyscope/pull/35 wakatime

**Comment by BoBeR182** [comment]
https://github.com/SpectralOps/keyscope/pull/34 opsgenie

**Comment by BoBeR182** [comment]
https://github.com/SpectralOps/keyscope/pull/38 lokalize
